src/disnake/ext/plugins/plugin.py

```diff
@@ -258,16 +321,25 @@ def __init__(
         slash_command_attrs: t.Optional[SlashCommandParams] = None,
         user_command_attrs: t.Optional[AppCommandParams] = None,
         logger: t.Union[logging.Logger, str, None] = None,
+        **extras: t.Any,
     ) -> None:
         self.metadata: PluginMetadata = PluginMetadata(
             name=name or _get_source_module_name(),
-            category=category,
             command_attrs=command_attrs or {},
             message_command_attrs=message_command_attrs or {},
             slash_command_attrs=slash_command_attrs or {},
             user_command_attrs=user_command_attrs or {},
+            extras=extras,
         )
 
+        if category is not None:
+            warnings.warn(
+                "Passing `category` to `Plugin` is deprecated. Use `extras` instead.",
+                DeprecationWarning,
+            )
+
+            self.metadata.extras["category"] = category
+
```

Comment:
This is pretty pointless. After deprecation of the category keyword-arg, it'll automatically be picked up by **extras anyways.

Reply:
Clever point, thanks! 451310ccbc09b1dcf20091cef3268523f47f6e51